renga/models/dataset.py

```diff
@@ -208,25 +198,28 @@ def add_data(self, url, datadir=None, git=False, **kwargs):
 
         if git:
             if isinstance(target, (str, NoneType)):
-                self.files.update(self._add_from_git(self.path, url, target))
+                self.files.update(
+                    self._add_from_git(repo, self.path, url, target))
             else:
                 for t in target:
-                    self.files.update(self._add_from_git(self.path, url, t))
+                    self.files.update(
+                        self._add_from_git(repo, self.path, url, t))
         else:
-            self.files.update(self._add_from_url(self.path, url, **kwargs))
+            self.files.update(
+                self._add_from_url(repo, self.path, url, **kwargs))
 
-    def _add_from_url(self, path, url, nocopy=False, **kwargs):
+    def _add_from_url(self, repo, path, url, nocopy=False, **kwargs):
```

Comment:
don't think `repo` is needed here at all

Reply:
We do in https://github.com/SwissDataScienceCenter/renga-python/pull/123/files#diff-ceb5f0488f1df82397418d3727d4eb35R231

Reply:
but that's recursing, we can remove it in both places